lib/strategies/simple-arbitrage.js

```diff
@@ -118,13 +123,16 @@ function largestSpreadArbitrage(pair, time) {
     pdata.volumeNeeded    = ((parseFloat(highest_bid) + parseFloat(lowest_ask)) * low_amt);
     pdata.percentProfit   = (pdata.totalProfit/pdata.volumeNeeded);
 
-    outputPerformanceData(data,pdata);
+    outputPerformanceData(exchSumm, pdata);
+    if (pdata.totalProfit > 0){
+      return (order);
+    }
+    return(null);
 
-  }, time);
 }
 
 // Export
 module.exports = {
-  largestSpreadArbitrage,
+  strategy,
   order
 }
```

Comment:
I changed the name of the driving function to standardize calls to strategies, I'll add a comment